tests/cli/cli_test.go

```diff
@@ -89,7 +92,7 @@ func run(t testing.TB, commands []Command) {
 
 		b, err := cli.Command(args...).CombinedOutput()
 		t.Log(fmt.Sprintf("\n$ %s\n%s", cmd.Command, string(b)))
-		if err != nil {
+		if !shouldError && err != nil {
```

Comment:
What if we take advantage of `Command.Output` being an `interface{}`. Example:

```go
run(t, []Command{
        {
                "deploy remind101/acme-inc",
                errors.New("error: A message is required for this action, please run again with '-m'."),
        },
})
```

Then, we can use that to test whether a test should expect an error:

```go
if expectedErr, ok := cmd.Output; ok {
        if err != expectedErr {
                t.Fatalf("Expected %v, got %v", err, expectedErr)
        }
} else {
        if err != nil {
                t.Fatal(err)
        }
}
```